src/settings/template.ts

```diff
@@ -198,7 +198,7 @@ export const renderArticleContnet = async (
     return {
       text: formatHighlightQuote(highlight.quote, template),
```

Comment:
Could you return raw text for highlighted content with images?

Reply:
@wuminbin May I know why do you want the raw text for highlighted content with images?

We probably could make it an option to choose raw text versus full markdown